python/test/unit/language/test_core.py

```diff
@@ -3235,6 +3235,11 @@ def test_dot3d(B, num_warps, M, N, K, in_dtype_str, out_dtype_str, device):
     if is_hip():
         # hip does not support tf32 precision, so use ieee for all tests
         input_precision = "ieee"
+        if triton.runtime.driver.active.get_current_target().arch == "gfx1100":
+            if in_dtype_str == "int8" or in_dtype_str == "float32":
+                pytest.skip(f"{in_dtype_str} is not supported in WMMA dot")
+            if out_dtype_str == "float16":
```

Comment:
I need to check this. At some point they did not match, but maybe this is not the case anymore, since a lot of time passed since I've implemented this.

Reply:
Yes, precision issue is still there.
I suspect this is a hardware problem, though this requires more investigation.

Reply:
Okay thanks. worth understanding more. I think we can also prmote to f32 and then downcast if necessary.

Reply:
@binarman Are we currently using `V_WMMA_F16_16X16X16_F16` and see accuracy mismatch with pytorch? If so, can we use `V_WMMA_F32_16X16X16_F16` and then cast to fp16 as @antiagainst mentioned?